devito/jit_manager.py

```diff
@@ -20,6 +20,7 @@ class JitManager(object):
     _hashing_function = sha1
     _wrapped_functions = None
     COMPILER_OVERRIDE_VAR = "DEVITO_CC"
+    REMOVE_FLAG_VAR = "DEVITO_REMOVE_FLAG"
```

Comment:
This is fine for now, but we might want to consider going back to providing our own compiler-specific default flags, rather than force-removing things from the Python compilation settings.